src/graph/graphQL.ts

```diff
@@ -18,11 +18,16 @@ const schema = require("./schema.cortex.json");
  * Note: Use __dirname to get the current directory of the calling script.
  * @param {string} path
  * @param {string} current
+ * @param {{[p: string]: string | boolean | number}} parameters
  * @returns {string}
  */
-export function subscriptionFromFile(path: string, current: string = appRoot.path): string {
+export function subscriptionFromFile(path: string,
+                                     current: string = appRoot.path,
+                                     parameters: {
+                                        [name: string]: string | boolean | number;
+                                     } = {}): string {
```

Comment:
Or perhaps this is the actual function that is called everywhere?

Reply:
Yeah, that is the only function we call.